lib/static/modules/utils/index.js

```diff
@@ -210,17 +210,34 @@ export function parseKeyToGroupTestsBy(key) {
     return [groupSection, groupKey];
 }
 
-export function getBlob(url) {
-    return new Promise((resolve, reject) => {
-        const xhr = new XMLHttpRequest();
+export async function getBlobWithRetires(url, retriesCount = 3) {
```

Comment:
We can also be rps limited
So exponential retries (1s, 2s, 4s) would not be superfluous